### ISSUE TITLE
If I want to train from scratch ,what kind of data should be prepared?

### ISSUE THREAD

**Issue by liuchangf** [Lydorn/Polygonization-by-Frame-Field-Learning#19]
Hello There:

Great Job! A novel idea in Polygonal Building Extraction as I know .
In traditional segmentation , training data include two kinds of data ,one is image ( usually is RGB image) ,another is one channel label( the same size with the image ,but only one channel ,every pixel store a number for a certain class). 
In your case (this repo) ,I'm confusing what kind of data should be prepared , there is no doubt that image will be one of them,but how about others ,there is no details in your paper.
![1](https://user-images.githubusercontent.com/31433358/123054641-a1eebe80-d437-11eb-988d-a313f805096b.PNG)
In readme, I find the dir structe , I think images is for images just like segmentation, but how about gt?
Is this gt is same as label in the segmentation? what is ge polygonized? I found there is all *.geojson files in the folder ,what are they for ? I found *.npy files in aligned_gt_polygons_2 and gt_polygons , what are these files for ?
Waiting for your kindly reply , than you in advance!

**Comment by mohammadreza-sheykhmousa** [comment]
@Lydorn  Hi Nicolas, well done with the paper and I am sure you successfully managed to defend your PhD. 
Regarding my problem, I have the same questions as @liuchangf. Would you please shed some light on this topic?

My specific question is about:

- What should be the structure of Geojson  file for roof footprint segmentation? 
- Can i sue only one labeled building as an input and not multiple building?
- Can I extend your model and train/predict on roof instance segmentaion -in a case that my gt is comprised one building and related instances of that building?

I greatly appreciate your reply! 
Regards, Reza

**Comment by franz-waldner** [comment]
Also intersted in an answer 
:)
